cloud_recording_list_all.go

```diff
@@ -0,0 +1,62 @@
+package zoom
+
+import "fmt"
+
+const (
+	// ListAllRecordingsPath - v2 lists all recordings
+	ListAllRecordingsPath = "/users/%s/recordings"
+
+	// TrashTypeMeetingRecordings list all meeting recordings from the trash. Default.
+	TrashTypeMeetingRecordings TrashType = "meeting_recordings"
+	// TrashTypeRecordingFile list all individual recording files from the trash
+	TrashTypeRecordingFile TrashType = "recording_file"
+)
+
+type (
+	// TrashType is the type of Cloud recording that you would like to retrieve from the trash
+	TrashType string
+
+	// ListAllRecordingsOptions contains options for ListAllRecordings.
+	ListAllRecordingsOptions struct {
+		UserID        string `url:"-"`
+		PageSize      *int   `url:"page_size,omitempty"`
+		NextPageToken string `url:"next_page_token,omitempty"`
+		Mc            string `url:"mc,omitempty"`
+		Trash         bool   `url:"trash,omitempty"`
+		// From is a YYYY-MM-DD string representing a date
+		From string `url:"from"`
```

Comment:
The query URL parser doesn't like non `time.Time` fields. It just ignores this field if it's a zoom.Date or a zoom.Time.

Reply:
👍